### PR TITLE
feat: network selector on explorer

### LINKED ISSUE #4886: Explorer Network switch
Being done in 
- https://github.com/kodadot/nft-gallery/pull/4863

### PR DESCRIPTION
**Pull request by preschian**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

- [x] Closes #4886
- [x] branching from https://github.com/kodadot/nft-gallery/pull/4846
- [x] related with #4474

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=DY4SQF2iD456tH89aQtz5wv1EV3BbSW8wKKuMcwbmXaj1pM)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

image

### CODE REVIEW

**Comment by yangwao** [comment]
powerful

**Comment by exezbcz** [comment]
Hiii, nice
one little change from the sort by issue
- text to black - on k-accent(primary) we dont use white color because there is smaller contrast than with black.
image

can we please make the button work as in gallery chart; when you choose sub-option it became visible on the button. 
so when I chose basilisk 
image
- the button should have the text - basilisk

Will we enable multi-select like on sort by? (not sure in this stage) If so, the number of activated options would be useful as well. cc @yangwao 
Thanks!

**Review by roiLeo — COMMENTED** [review]
> in the current stage, the network is already selected how about change the text to this for now: `Network: Basilisk`

^ yes! 

at the moment it's confusing to have written "All Networks" while only one is showing/selected

**Comment by exezbcz** [comment]
can we make it to only display the network name without the "network:" 
- now the button is changing its width based on how long the name is - can we make it fixed? 
image
otherwise alright! thanks

**Comment by roiLeo** [comment]
> now the button is changing its width based on how long the name is - can we make it fixed?

are we sure about fixed width? it can have an impact on responsive mode

File: components/explore/ExploreChain.vue [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+function onSwitchChain(chain) {
+  $store.dispatch('setUrlPrefix', chain)
+
+  router.push({
+    params: {
+      prefix: chain,
+    },
+    query: {
+      page: '1',
+      sort: route.query.sort,
```
Comment (Jarsen136) [inline components/explore/ExploreChain.vue:43]: how about adding the search keywords to query

**Comment by yangwao** [comment]
> Will we enable multi-select like on sort by?

probably not in this

> how about change the text to this for now: `Network: Basilisk`

yes, could be 

Terms of stuff in button, I would keep for sake clarity end user "Network" there as prefix, yet on dropdown could be w/o

[rarible](https://rarible.com/explore/all/items?multiple=all&verified=false&blockchains[]=POLYGON&sort=latest)
image

**Comment by yangwao** [comment]
btw, sry disturbing lovely discussion, yet happy to move on and not get stuck on wording :D

**Comment by yangwao** [comment]
> can we make it to only display the network name without the "network:"

we can still use some sort of label. People outside of Ethereum barely knows what are names of other parachains in Polkadot, not even canary networks on Kusama

**Comment by yangwao** [comment]
> what are the other options how we can avoid button jumping? - there will be a layout change button in future - which would be influenced by this.

we can reflect in explorer feedback

**Comment by yangwao** [comment]
pay 50 usd

**Pull request merged**